packages/owl-bot/test/owl-bot.ts

```diff
@@ -46,8 +41,9 @@ describe('owlBot', () => {
         }),
       },
     });
-    probot.load((app: Probot) => {
-      owlBot('abc123', app);
+    await probot.load((app: Probot) => {
```

Comment:
Tests were failing in isolation without mocking firestore, I've opted to provide it as a plain stub for now, when we eventually test its functionality, I will make a helper insted.